src/services/chris_integration.tsx

```diff
@@ -212,6 +212,18 @@ class ChrisIntegration {
     return (await client.getPACSFiles({ fname_exact: imgTitle })).data
   }
 
+  static async getFilePathNameByUID(StudyInstanceUID: string, SeriesInstanceUID: string): Promise<string> {
+    let client: any = await ChrisAPIClient.getClient();
+    
+    const res = await client.getPACSFiles({
+      StudyInstanceUID,
+      SeriesInstanceUID,
+      limit: 1
```

Comment:
I propose creating a file containing constants such as pagination limit and offsets.